invenio_rdm_records/records/mappings/v6/rdmrecords/drafts/draft-v4.0.0.json

```diff
@@ -325,19 +325,33 @@
                 }
               }
             },
-            "licenses" : {
+            "rights" : {
+              "type": "object",
               "properties" : {
-                "identifier" : {
-                  "type" : "keyword"
+                "@v": {
+                  "type": "keyword"
                 },
-                "license" : {
-                  "type" : "text"
+                "id": {
+                  "type": "keyword"
                 },
-                "scheme" : {
-                  "type" : "keyword"
+                "title": {
+                  "type": "object",
+                  "dynamic": true
                 },
-                "uri" : {
-                  "type" : "keyword"
+                "description" : {
+                  "type" : "object",
+                  "dynamic": true
+                },
+                "props": {
+                  "type": "object",
+                  "properties": {
+                    "url": {
+                      "type": "keyword"
+                    }
+                  }
+                },
+                "link": {
+                  "type": "keyword"
```

Comment:
If we are not doing searches on this field (I doubt it 🤔 ?) it can be `index: false`